kolibri/plugins/coach/assets/src/views/common/status/CoachStatusIcon.vue

```diff
@@ -33,8 +27,13 @@
       },
     },
     computed: {
-      ICONS() {
-        return ICONS;
+      iconType() {
+        return {
+          [ICONS.clock]: 'inProgress',
+          [ICONS.star]: 'mastered',
+          [ICONS.help]: 'helpNeeded',
+          [ICONS.nothing]: 'notStarted',
+        }[this.icon];
```

Comment:
We should probably just get rid of `CoachStatusIcon` and these special-case `ICONS` constants, and them with `KIcon`? Could do this in a second pass to keep the PR simpler